### PR TITLE
More BF stuff

### PR DESCRIPTION
**Pull request by strengejacke**

### CODE REVIEW

File: R/bayesfactor_inclusion.R [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+
+    for (eff in effnames) {
+      df.interaction[,eff] <- sapply(effnames, function(x) includes_interaction(x,eff))
+    }
+    rownames(df.interaction) <- effnames
+    df.interaction <- df.interaction[,-1]
+  }
+
+  # Build Effect Table #
+  df.effect <- data.frame(effnames,
```
Comment (strengejacke) [inline R/bayesfactor_inclusion.R]: effnames might be a factor, unless you set `stringsAsFactors = FALSE` here. Could you check if this is working as intended?

File: R/bayesfactor_inclusion.R [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+      rand_print[[i]] <- paste0("(", tmp_trms, "|", random_units[[i]], ")")
+    }
+
+    stats::setNames(
+      c(fix_trms,unlist(rand_trms)),
+      c(fix_trms,unlist(rand_print))
+    )
+  }
+
+  print_terms <- character()
```
Comment (strengejacke) [inline R/bayesfactor_inclusion.R]: `print_terms` is defined but not used...

File: R/bayesfactor_inclusion.R (hunk 1 above)
Comment (mattansb) [reply R/bayesfactor_inclusion.R]: Why oh why is `stringsAsFactors = TRUE` by default?

Comment (mattansb) [reply R/bayesfactor_inclusion.R]: It works any way, but I set `stringsAsFactors = FALSE` just to be safe.

File: R/bayesfactor_inclusion.R (hunk 2 above)
Comment (mattansb) [reply R/bayesfactor_inclusion.R]: removed

**Comment by strengejacke** [comment]
Can we have a merge into master once this somehow works? Finetuning can be added later...

**Comment by mattansb** [comment]
All `bayesfactor_*` functions have been tested to work, so I think it's safe to merge them into the master.

**Comment by strengejacke** [comment]
> Pull requests that have a failing status can’t be merged on a phone.

Haha... Ok, must go to my PC now.

**Pull request merged**